README.md

```diff
@@ -40,32 +40,56 @@ export BITBUCKET_TOKEN=my_bitbucket_app_password
 
 - Check this repo out, and put it in the path.
 
-```bash
-bsh ❯ bb-pr
+```shell
 
-Simple tooling that helps management of bitbucket pull requests from
-the commandline
+Tool that helps management of bitbucket pull requests from the commandline
 
-Usage: bb-pr [help|list|squash-msg|squash-merge|approve|unapprove|decline] [options]
+Usage: bb-pr [help|list|squash-msg|squash-merge|approve|unapprove|decline|close-branch] [options]
   help         : show this help
   list         : list (open) PRs in this repo
   squash-msg   : copy a reasonable message to the clipboard for merging a PR
   squash-merge : merge the PR using the message from 'squash-msg'
   approve      : approve a PR (though should you from the CLI?)
   unapprove    : remove your approval
   decline      : decline a PR
+  close-branch : change the 'close_source_branch' field
 
-'squash-msg' | 'squash-merge' | 'approve' | 'unapprove' | 'decline'
+'squash-msg' | 'squash-merge' | 'approve' | 'unapprove' | 'decline' | 'close-branch'
```

Comment:
Does the name close-branch make sense? Because it implies it might close  the branch.

Not sure on better name, thought, just a thought.

Its probably OK.